apis/kueue/v1beta1/workload_types.go

```diff
@@ -204,6 +204,17 @@ type WorkloadStatus struct {
 	// +patchMergeKey=name
 	// +kubebuilder:validation:MaxItems=8
 	AdmissionChecks []AdmissionCheckState `json:"admissionChecks,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
+
+	// desiredResources provides a detailed view of the resources that were
+	// requested by a non-admitted workload when it was considered for admission.
+	// If admission is non-null, desiredResouces will be empty because
+	// admission.resourceUsgae contains the detailed information.
+	//
+	// +optional
+	// +listType=map
+	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=8
+	DesiredResources []PodSetRequest `json:"desiredResources,omitempty"`
```

Comment:
Maybe desired suggests it should be a field in spec. How about names like: `RequestedResources`, or `ResourceRequests`?